lib/code_corps/model/user_invite.ex

```diff
@@ -0,0 +1,17 @@
+defmodule CodeCorps.UserInvite do
+  use CodeCorps.Model
+
+  @type t :: %__MODULE__{}
+
+  schema "user_invites" do
+    field :email, :string, null: false
+    field :role, :string, default: "contributor"
```

Comment:
This shouldn't have a default because it's possible for an invite to _not_ be for a project.